internal/interface/repositories/document_repository.go

```diff
@@ -44,10 +44,12 @@ func (r *documentRepository) FindAllDocuments(args *FindAllDocumentsArgs) (*[]en
 	err := r.db.Raw(`
 		SELECT *, documents.id AS document_id, users.id AS AuthorID 
 		FROM documents INNER JOIN users ON documents.user_id = users.id
-		WHERE documents.type_id LIKE ?
+		WHERE users.role_id LIKE ?
+		AND  documents.type_id LIKE ?
 		AND	 LOWER(documents.title) LIKE ?
 		AND  documents.created_at BETWEEN ? AND ?
 		OFFSET ? LIMIT ?`,
+		fmt.Sprintf("%%%s%%", strings.ToUpper(args.Organization)),
 		fmt.Sprintf("%%%s%%", strings.ToUpper(args.DocumentType)),
 		fmt.Sprintf("%%%s%%", strings.ToLower(args.Title)),
 		args.StartTime,
```

Comment:
(FindAllDocuments)
seems like`args.Organization` is input as `WHERE users.role_id LIKE ?`, am i missing something?

Reply:
i see, understand now lmao